src/Microsoft.Performance.SDK.Runtime/CustomDataSourceReference.cs

```diff
@@ -13,60 +13,55 @@ namespace Microsoft.Performance.SDK.Runtime
     /// <summary>
     ///     Creates a <see cref="AssemblyTypeReferenceWithInstance{T, Derived}"/> where T is <see cref="ICustomDataSource"/> and Derived is <see cref="CustomDataSourceReference"/>.
     /// </summary>
-    public sealed class CustomDataSourceReference
+    public abstract class CustomDataSourceReference
```

Comment:
The change log says this was made abstract to help with testing. But this interface change is greater than "you can not inherit the class," it also is "you cannot directly create the class." Since its public, anyone who was directly creating instances is now broken.

Why not just make this a `public class`?